src/components/map/MapPage.js

```diff
@@ -46,7 +46,7 @@ const MapPage = () => {
 
   useEffect(() => {
     async function fetchClusterAndLocationData() {
-      if (view.bounds) {
+      if (view.bounds?.ne.lat) {
```

Comment:
Weird edge case, but in South America or Africa this value could be zero and thereby false-y.

Reply:
Good point, better to be safe here.